gox/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=gox
+pkg_version=master
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache 2")
+pkg_source=https://github.com/mitchellh/gox/archive/master.tar.gz # I'm killing a kitten.
+pkg_upstream=https://github.com/mitchellh/gox
+pkg_shasum=8eca6712394ce222afca49703f548651c42a8bb95303ac85926833cb47707f9a
+pkg_deps=(core/go core/gcc-libs core/glibc)
```

Comment:
The compiled gox binary appears to have no external dependencies.

core/go should be moved to pkg_build_deps and pkg_deps should be removed completely, since there's no runtime deps.